cds_dojson/marc21/fields/base.py

```diff
@@ -41,47 +40,6 @@ def agency_code(self, key, value):
     return 'SzGeCERN'
 
 
-@model.over('report_number', '^(037|088)__')
-@for_each_value
-def report_number(self, key, value):
```

Comment:
I wonder if this was thought only for videos (cos I saw that you moved to a specific videos part) or for any `future` generic record.

Reply:
Same applies to the other methods.

Reply:
I've reorganised it, because there were some rules in conflict with books data model. For example, in books we don't have contributors field but authors, with different roles, so output field was different. Also the rule for books report_numbers has additional checks of values in comparison with videos, that's why they could not be kept as common for every model. Probably before there was not enough information to predict that it won't work in every case.